content/200-concepts/150-data-platform/08-billing/02-plans-and-quotas.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Plans and quotas'
+metaTitle: 'Plans and quotas'
+metaDescription: 'The Prisma Data Platform provides a set of monthly plans that include quotas for the number of collaborators and environments per project, as well as a quota for the usage of the Prisma Data Proxy (in CPU milliseconds).'
+---
+
+<TopBlock>
+
+The Prisma Data Platform provides a set of monthly plans that include quotas for:
+
+- number of collaborators per project
+- number of environments per project
+- Prisma Data Proxy usage (in CPU milliseconds) per project
+
+</TopBlock>
+
+## About plans and quotas
+
+Plans in the Prisma Data Platform are designed with the goal to help you scale your projects as the need for team members, environments, and Data Proxy usage grows.
+
+Because of this, a single Prisma Data Platform plan applies for a single project.
+
+We want to give you the option to create projects freely and you can then decide which project you want to upgrade to a paid plan. You can have 100 Prisma projects and you can upgrade only 2 of them to paid plan.
+
+<Admonition>
+
+**Account-wide plans**<br /><br />
+The notion of an account-wide plan that includes a certain number of projects or provides access to specific features does not exist in the Prisma Data Platform.<br />
+
+</Admonition>
+
+## The first plan of each project
+
+- If a project is created or added in the Prisma Data Platform before the [General Availability](/about/prisma/platform-releases#maturity-levels) of the platform, it starts on the `Early Adopter` plan.
+- If project is created after General Availability, it starts on the `Free` plan.
+
+## Comparison of plans
```

Comment:
I would not feature the EA plan here, but rather move it into its own page. At some point, this page could then be removed.

Reply:
EA plan page is now created. Content is pushed to there and links added.